api/envoy/service/load_stats/v2/lrs.proto

```diff
@@ -66,8 +66,14 @@ message LoadStatsRequest {
 // [#not-implemented-hide:] Not configuration. TBD how to doc proto APIs.
 message LoadStatsResponse {
   // Clusters to report stats for.
+  // Not populated if *send_all_clusters* is true.
   repeated string clusters = 1 [(validate.rules).repeated = {min_items: 1}];
 
+  // If true, the client should send all clusters it knows about.
+  // Only clients that advertise the "envoy.lrs.supports_send_all_clusters" capability in their
+  // :ref:`client_features<envoy_api_field_core.Node.client_features>` field will honor this field.
+  bool send_all_clusters = 4;
```

Comment:
I think this should be v3-only, but I imagine this is going to cause you folks pain? @envoyproxy/api-shepherds

Reply:
I'm going to have to recuse myself and ask the other @envoyproxy/api-shepherds to weigh in on whether an exception can be made here.

Reply:
@lizan can we get your opinion on this one as well? Thanks.